Userland/Utilities/unzip.cpp

```diff
@@ -124,7 +127,25 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     }
 
     auto success = zip_file->for_each_member([&](auto zip_member) {
-        return unpack_zip_member(zip_member, quiet) ? IterationDecision::Continue : IterationDecision::Break;
+        bool keep_file = false;
+
+        if (file_filters.size() != 0) {
+            for (auto& filter : file_filters) {
+                // Convert underscore wildcards (usual unzip convention) to question marks (as used by StringUtils)
+                String string_filter = AK::StringUtils::replace(filter, "_", "?");
```

Comment:
```suggestion
                auto string_filter = filter.replace("_", "?", true);
```
You don't need to use `StringUtils` directly, this is mostly used to have identical implementations for methods of `String` and `StringView`. Also, use `True` to replace all occurences of `_` in `filter`.